### PR TITLE
AM-467-JUnit-Integration-Tests

### PR DESCRIPTION
**Pull request by nitinsahu1**
AM-464: Create testing framework and skeleton for new micro-service
Task : AM-467 Test setup for JUnit Tests and Integration Tests for ccd-case-document-api service

##JIRA link (if applicable) ##
https://tools.hmcts.net/jira/browse/AM-467

### Change description ###
AM-467 Test setup for JUnit Tests and Integration Tests for ccd-case-document-api service
Note : As discussed, only required files and code has been pushed as part of this PR to keep changes separately.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -18,6 +18,7 @@ plugins {
   id 'org.owasp.dependencycheck' version '5.2.4'
```
Comment (aashishaggarwal123) [inline build.gradle]: Can you please remove all the changes which are not related to Unit and integration testing elements.

File: build.gradle [hunk 2]
```diff
@@ -26,14 +27,14 @@ apply plugin: 'project-report'
...
-  junit              : '5.5.0',
+  junit           : '5.5.0',
   lombok             : '1.18.6',
   gradlePitest       : '1.3.0',
   pitest             : '1.4.2',
   reformHealthStarter: '0.0.5',
   reformLogging      : '5.1.0',
   reformS2sClient    : '3.0.0',
-  serenity           : '2.0.91',
+  serenity           : '2.0.23',
```
[the remark this answers is not in the document]
Comment (nitinsahu1) [reply build.gradle]: Its updated.

File: build.gradle [hunk 3]
```diff
@@ -65,6 +65,13 @@ jacocoTestCoverageVerification {
   }
 }
 
+configurations {
```
[the remark this answers is not in the document]
Comment (nitinsahu1) [reply build.gradle]: Its aligned to Integration test and only Integration test specific changes should be merged to upstream branch.

File: build.gradle (hunk 1 above)
Comment (nitinsahu1) [reply build.gradle]: Added comments in build.gradle file for only changes related to Integration tasks.
